neo/Consensus/ConsensusContext.cs

```diff
@@ -349,6 +350,7 @@ public void Reset(byte viewNumber)
                 };
                 var pv = Validators;
                 Validators = NativeContract.NEO.GetNextBlockValidators(Snapshot);
+                PrepareRequestThreshold = TimeSpan.FromMilliseconds((Validators.Length + 1) * Blockchain.TimePerBlock.TotalMilliseconds);
```

Comment:
What about `8 * Blockchain.TimePerBlock.TotalMilliseconds`? I can't see any relationship between the number and the threshold.

Reply:
My rationale is about how much time a malicious node can earn per round, so it depends the number of the CN.

Reply:
https://github.com/neo-project/neo/blob/72be7f6a26f3ed222f1ad5dc1b4391c62fa4f67d/neo/Consensus/ConsensusService.cs#L408

But it is compared to the current time, not the current block time. So you can't earn time from every round.

Reply:
Agree, so.. What do you prefer: fix it to 2 minutes, or 8 * blockTime?

Reply:
Both are good to me.